src/services/ticketing-service/index.ts

```diff
@@ -136,6 +139,36 @@ export const routes = (router: KoaRouter) => {
     }
   })
 
+  router.get('(.*)/maintenanceUnits/:rentalPropertyId/:type?', async (ctx) => {
+    try {
+      const maintenanceUnits = await getMaintenanceUnitsForRentalProperty(
+        ctx.params.rentalPropertyId
+      )
+      if (maintenanceUnits && maintenanceUnits.length > 0) {
+        // Filter by type if type is provided
+        if (ctx.params.type) {
+          ctx.status = 200
+          ctx.body = maintenanceUnits.filter(
+            (unit) => unit.type.toUpperCase() === ctx.params.type.toUpperCase()
+          )
```

Comment:
```suggestion
          ctx.body = {
            content:  maintenanceUnits.filter(
               (unit) => unit.type.toUpperCase() === ctx.params.type.toUpperCase()
          )}
```
"Vi" (jag ser att jag missat det typ överallt) har tidigare returnerat ett data objekt à la `ctx.body = { data: responseData }`, men under senaste devForum så beslöt vi oss för att ändra `data` till `content`.

Ser att det saknas på många i den här filen men lika bra att börja någonstans.